reference/supported-languages.md

```diff
@@ -0,0 +1,36 @@
+# Languages/Formats OpenRewrite supports
+
+OpenRewrite is continuously adding support for new languages and formats. As of June 2023, OpenRewrite supports:
+
+## Programming languages
+
+* Java
+* Groovy/Gradle 
+* Kotlin
+* COBOL (open-source, but not free for commercial use)
+
+## Data formats
+
+* XML
+* Properties
+* YAML
+* JSON
+* Protobuf
+* Maven POMs (with dependency management)
```

Comment:
We could pull Maven and Gradle into a separate block, with Bazel soon to be added, and possibly tools like NPM as well.
```suggestion

## Build tools
* Maven
* Gradle
```